docs/install-upgrade/config.md

```diff
@@ -40,45 +21,165 @@ access the configured targets. It could be done by passing `--control-proxy=true
+        controlProxyURL: http://172.30.1.1:31028
+        lokiTargets:
+          grafana_cloud: # target name, multiple targets can be configured
+              basicAuth: # optional
+                  password: "<password>"
+                  username: "<username>"
+              labels: # labels to be added to all logs
+                  env: env-1
+              url: https://logs-prod-021.grafana.net/loki/api/v1/push
+              useControlProxy: true # if the Loki API is not available from the switches directly, use the Control Node as a proxy
+        prometheusTargets:
+          grafana_cloud: # target name, multiple targets can be configured
+              basicAuth: # optional
+                  password: "<password>"
+                  username: "<username>"
+              labels: # labels to be added to all metrics
+                  env: env-1
+              sendIntervalSeconds: 120
+              url: https://prometheus-prod-36-prod-us-west-0.grafana.net/api/prom/push
+              useControlProxy: true # if the Loki API is not available from the switches directly, use the Control Node as a proxy
+              unixExporterCollectors: # list of node-exporter collectors to enable, https://grafana.com/docs/alloy/latest/reference/components/prometheus.exporter.unix/#collectors-list
+                  - cpu
+                  - filesystem
+                  - loadavg
+                  - meminfo
+              collectSyslogEnabled: true # collect /var/log/syslog on switches and forward to the lokiTargets
 ```
 
 For additional options, see the `AlloyConfig` [struct in Fabric repo](https://github.com/githedgehog/fabric/blob/master/api/meta/alloy.go).
+
+### Configure switch users
+
+It's currently only possible by using a yaml configuration file for the `hhfab init -c <config-file.yaml>` command. You can specify users to be configured on the switches in the following format:
+```yaml
+spec:
+    config:
+      control:
+        defaultUser: # user 'core' on all control nodes
+          password: "hashhashhashhashhash" # password hash
+          authorizedKeys:
+            - "ssh-ed25519 SecREKeyJumblE"
+
+        fabric:
+          mode: spine-leaf # "spine-leaf" or "collapsed-core"
+          
+          defaultSwitchUsers: 
+            admin: # at least one user with name 'admin' and role 'admin'
+              role: admin
+              #password: "$5$8nAYPGcl4..." # password hash
+              #authorizedKeys: # optional SSH authorized keys
+              #  - "ssh-ed25519 AAAAC3Nza..."
+            op: # optional read-only user
+              role: operator
+              #password: "$5$8nAYPGcl4..." # password hash
+              #authorizedKeys: # optional SSH authorized keys
+              #  - "ssh-ed25519 AAAAC3Nza..."
+
+```
+The role of the user,`operator` is read-only access to `sonic-cli` command on the switches. In order to avoid conflicts, do not use the following usernames: `operator`,`hhagent`,`netops`.
```

Comment:
```suggestion

The role of the user `operator` is read-only access to `sonic-cli` command on the switches.

In order to avoid conflicts, do not use the following usernames: `operator`, `hhagent`, or `netops`.
```

The last part might be worth a warning call-out?